tests/unit/build_test.py

```diff
@@ -103,3 +121,44 @@ def test_build_container_invalid_container_limits(self):
                 'foo': 'bar'
             })
         )
+
+    def test__set_auth_headers_with_empty_dict_and_auth_configs(self):
```

Comment:
Small thing, but there's a double underscore after `test`, which we don't do anywhere else. Same thing with the other new tests.